lighthouse-core/config/seo.js

```diff
@@ -28,7 +37,8 @@ module.exports = {
       description: 'These ensure your app is able to be understood by search engine crawlers.',
       audits: [
         {id: 'meta-viewport', weight: 1, group: 'seo-mobile'},
-        {id: 'document-title', weight: 1, group: 'seo-content'}
+        {id: 'document-title', weight: 1, group: 'seo-content'},
+        {id: 'meta-description', weight: 1, group: 'seo-content'}
```

Comment:
nit: go for the trailing comma here so future diffs will be cleaner

Reply:
I saw trailing comma being used here and there, but since there is no 100% consistency, I left it out. The cleaner diffs argument SGTM, maybe we should make eslint to check for that?

Reply:
AFAIK there wasn't any effort to do trailing commas other than occasionally sneaking them in as we touched code but huge +1 to turning on the eslint rule :)

Reply:
I've opened an issue for it: https://github.com/GoogleChrome/lighthouse/issues/3304 👍